yarn-project/noir-contracts/src/contracts/noir-aztec3/src/abi.nr

```diff
@@ -54,6 +56,30 @@ impl CallContext {
     }
 }
 
+global COMMITMENT_TREES_ROOTS_SIZE: comptime Field = 4;
+
+struct CommitmentTreesRoots {
+    old_private_data_tree_root : Field,
+    old_nullifier_tree_root : Field,
+    old_contract_tree_root : Field,
+    old_l1_to_l2_messages_tree_root : Field,
+}
+
+impl CommitmentTreesRoots {
+    fn serialize(self) -> [Field; COMMITMENT_TREES_ROOTS_SIZE] {
+        [
+            self.old_private_data_tree_root,
```

Comment:
The ordering in here differs from the ordering in the executor.ts Why? Also, do they need the old_ prefix, we are not using that most other places as the roots are mainly for historic.